### ISSUE TITLE
Chat should open at/navigate to "new messages" marker

### ISSUE THREAD

**Issue by Flohack74** [ubports/telegram-app#76]
When a chat is opened that has new messages inside, the new messages marker is not always visible and one has to scroll back. Could be a timing issue. Maybe best is to prevent scrolling until the user has reached the end for the first time after opening the window or IDK.

**Comment by mymike00** [comment]
sometimes it happened to me that when I opened a chat it crolled correctely to the "new messages" marker but then some images/media were loaded so the chat scrolled and it was not on the marker...

**Comment by Flohack74** [comment]
Yes this I know. But the greater deal is that it only scrolls to new messages marker if there are max 50 new messages,. this is what gets loaded. I need to re-think the loading a bit.

To stabilize the window when media is loading is beyond my skills probably ;)

**Comment by mymike00** [comment]
yes, I know about the 50 messages.
maybe the app could only wait for the media loading, but it will wait to much if there are lots of messages...

**Comment by Flohack74** [comment]
I try now to load n messages where n = min(50, *count_of_new_msg*) - lets test this a bit, should be better now. Unfortunately the blue marker is now sometimes right *above* the first emssage, we need to maybe load n + 4 or so messages ;)

**Comment by mymike00** [comment]
Yeah, but anyway if you wait a bit, the messages before the number n are loaded and with them also the marker is loaded
